scripts/setup-adapters.sh

```diff
@@ -35,7 +35,13 @@ function install_aws_deps {
   # We need this specific version of Minio for testing.
   if [[ "$OSTYPE" == linux-gnu* ]]; then
     wget https://dl.min.io/server/minio/release/linux-amd64/archive/minio-20220526054841.0.0.x86_64.rpm
-    rpm -i --replacepkgs minio-20220526054841.0.0.x86_64.rpm
+    if yum list installed minio >/dev/null 2>&1;then
+      if  prompt "Do you want to replace minio package with compatible version minio-20220526054841.0.0.x86_64.rpm?"; then
```

Comment:
let's say `Do you want to replace the installed Minio with version minio-20220526054841.0.0.x86_64.rpm needed for S3 tests?`

Reply:
Updated. Please review.